tests/controlboard/ControlBoardPositionControlTest.cc

```diff
@@ -0,0 +1,145 @@
+#include "../../libraries/singleton-devices/Handler.hh"
+#include <cmath>
+#include <cstdlib>
+#include <gtest/gtest.h>
+#include <gz/sim/Joint.hh>
+#include <gz/sim/Link.hh>
+#include <gz/sim/Model.hh>
+#include <gz/sim/TestFixture.hh>
+#include <gz/sim/Util.hh>
+#include <gz/sim/World.hh>
+#include <gz/sim/components/JointForceCmd.hh>
+#include <iostream>
+#include <string>
+#include <yarp/dev/IControlMode.h>
+#include <yarp/dev/IEncoders.h>
+#include <yarp/dev/IPositionControl.h>
+#include <yarp/dev/IPositionDirect.h>
+#include <yarp/dev/ITorqueControl.h>
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/os/Bottle.h>
+#include <yarp/os/BufferedPort.h>
+#include <yarp/os/Network.h>
+
+class ControlBoardPositionFixture : public testing::Test
+{
+protected:
+    // void SetUp() override
+    ControlBoardPositionFixture()
+        : testFixture{"../../../tests/controlboard/pendulum_joint_relative_to_parent_link.sdf"}
+    {
+        gz::common::Console::SetVerbosity(4);
+
+        testFixture.
+            // Use configure callback to get values at startup
+            OnConfigure([&](const gz::sim::Entity& _worldEntity,
+                            const std::shared_ptr<const sdf::Element>& /*_sdf*/,
+                            gz::sim::EntityComponentManager& _ecm,
+                            gz::sim::EventManager& /*_eventMgr*/) {
+                std::cerr << "========== Configuring test" << std::endl;
+
+                gz::sim::World world(_worldEntity);
+
+                // Get model
+                auto modelEntity = world.ModelByName(_ecm, "single_pendulum");
+                modelEntity = world.ModelByName(_ecm, "single_pendulum");
+                EXPECT_NE(gz::sim::kNullEntity, modelEntity);
+                model = gz::sim::Model(modelEntity);
+
+                driver = gzyarp::Handler::getHandler()->getDevice(deviceScopedName);
+                ASSERT_TRUE(driver != nullptr);
+                iPositionControl = nullptr;
+                ASSERT_TRUE(driver->view(iPositionControl));
+                iControlMode = nullptr;
+                ASSERT_TRUE(driver->view(iControlMode));
+                iEncoders = nullptr;
+                ASSERT_TRUE(driver->view(iEncoders));
+
+                // Get joint
+                auto jointEntity = model.JointByName(_ecm, "upper_joint");
+                EXPECT_NE(gz::sim::kNullEntity, jointEntity);
+                joint = gz::sim::Joint(jointEntity);
+
+                // Set joint in torque control mode
+                ASSERT_TRUE(iControlMode->setControlMode(0, VOCAB_CM_POSITION));
+
+                // Print number of joint configured
+                int nJointsConfigured{};
+                ASSERT_TRUE(iPositionControl->getAxes(&nJointsConfigured));
+                std::cerr << "Number of joints configured: " << nJointsConfigured << std::endl;
+
+                configured = true;
+                std::cerr << "========== Test configured" << std::endl;
```

Comment:
```suggestion
                std::cerr << "========== Test configured" << std::endl;
```

leftover? Or you want to leave it. If you want to leave it, I would at least remove the `"========== `